.pipelines/cni/k8s-e2e/k8s-e2e-job-template.yaml

```diff
@@ -6,8 +6,8 @@ parameters:
 
 
 jobs:
-  - job: cni_k8se2e
-    displayName: "CNI k8s E2E"
+  - job: CNI_${{ parameters.os }}
```

Comment:
I think this change, as well as the change in `.pipelines/cni/k8s-e2e/k8s-e2e-step-template.yaml`, should be in a separate PR

They don't seem related to dualstack overlay windows testing

Reply:
Hmm, just saw you add this into the dualstack step
Is there a reason why?

Reply:
it will run windows conformance tests for dualstack

Reply:
This is because there are 2 calls of this template in a single stage. This requires a the `job:` field to be unique for multiple calls and not possible without giving it parameter